### PR TITLE
Add cleos system activate subcommand (to activate system feature like kv_database)

### PR DESCRIPTION
**Pull request by softprofe**
## Change Description

Start 10036 description ......

EPE 453
Add cleos system activate subcommand (to activate system feature like kv_database)
you can find usage by:   cleos system activate -h

support all system features.  support uppercase and lowercase   kv_database or KV_DATABASE or KV_database

                              PREACTIVATE_FEATURE
                              ONLY_LINK_TO_EXISTING_PERMISSION
                              FORWARD_SETCODE
                              KV_DATABASE
                              WTMSIG_BLOCK_SIGNATURES
                              REPLACE_DEFERRED
                              NO_DUPLICATE_DEFERRED_ID
                              RAM_RESTRICTIONS
                              WEBAUTHN_KEY
                              BLOCKCHAIN_PARAMETERS
                              DISALLOW_EMPTY_PRODUCER_SCHEDULE
                              ONLY_BILL_FIRST_AUTHORIZER
                              RESTRICT_ACTION_TO_SELF
                              CONFIGURABLE_WASM_LIMITS
                              ACTION_RETURN_VALUE
                              FIX_LINKAUTH_RESTRICTION
                              GET_SENDER

  using :  cleos system activate kv_database
  instead of  using  cleos push action eosio activate '["825ee6288fb1373eab1b5187ec2f04f6eacb39cb3a97f356a07c91622dd61d16"]' -p eosio

End 10036 description ......
## Change Type
**Select ONE**
- [ ] Documentation

- [ ] Stability bug fix

- [x] Other

- [ ] Other - special case

## Consensus Changes
- [ ] Consensus Changes

## API Changes
- [ ] API Changes

## Documentation Additions
- [x] Documentation Additions

new subcommand cleos system activate 
see usage by: cleos system activate -h
example:     cleos system activate kv_database
                   cleos system activate KV_DATABASE
                   cleos system activate KV_database
note :   feature name argument is not sensitive to uppercase or lowercase

### CODE REVIEW

File: programs/cleos/main.cpp [hunk 1]
```diff
@@ -2199,6 +2201,58 @@ struct closerex_subcommand {
...
+         string action="activate";
+         string data;
+         std::locale loc;
+         vector<std::string> permissions = {"eosio"};
+         for(auto & c : feature_name_str) c = std::toupper(c, loc);
+         if(map_name_digest.find(feature_name_str) != map_name_digest.end()){
+            std::string feature_digest = map_name_digest[feature_name_str];
+            data =  "[\"" + feature_digest + "\"]";
+         } else {
+            std::cout << "Can't find system feature : " << feature_name_str << std::endl;
```
Comment (linhuang-blockone) [inline programs/cleos/main.cpp:2243]: Exit when feature cannot be found.

File: programs/cleos/main.cpp [hunk 2]
```diff
@@ -2199,6 +2201,58 @@ struct closerex_subcommand {
...
+         }
+         fc::variant action_args_var;
+         if( !data.empty() ) {
+            action_args_var = json_from_file_or_string(data, fc::json::parse_type::relaxed_parser);
+         }
```
Comment (linhuang-blockone) [inline programs/cleos/main.cpp:2248]: Add an else to handle error.

**Review by linhuang-blockone — CHANGES_REQUESTED** [review]
* Is the command format agreed by Areg?
* Add command usage to Documentation section in the PR header

File: programs/cleos/main.cpp (hunk 1 above)
Comment (softprofe) [reply programs/cleos/main.cpp:2243]: > * Is the command format agreed by Areg?
> * Add command usage to Documentation section in the PR header

yes, It is Areg's idea to add subcommand under  cleos system.

Comment (linhuang-blockone) [reply programs/cleos/main.cpp:2243]: Click the documentation button in the PR and add documentation there.

Comment (softprofe) [reply programs/cleos/main.cpp:2243]: now documentation is there.

**Review by linhuang-blockone — APPROVED** [review]

**Pull request merged**